osu.Game.Rulesets.Tau/Localisation/SettingStrings.cs

```diff
@@ -26,6 +26,11 @@ public static class SettingStrings
         /// </summary>
         public static LocalisableString HitLighting => new TranslatableString(getKey(@"hit_lighting"), @"Hit Lighting");
 
+        /// <summary>
+        /// "Highlight Hard Beats"
+        /// </summary>
+        public static LocalisableString HighlightHardBeats = new TranslatableString(getKey(@"highlight_hard_beats"), @"Highlight Hard Beats");
```

Comment:
Add the corresponding string into Settings.resx too

Reply:
Would this string need to be added to all Settings.*.resx files or just Settings.resx?

Reply:
Just Settings.resx, crowdin manages the rest